### PR TITLE
add note about https://github.com/algolia/react-element-to-jsx-string

### PR DESCRIPTION
**Pull request by vvo**
I saw your comment in https://github.com/alexlande/react-to-jsx/pull/18#issuecomment-218600769, I thought I could do this for you.

Thanks again because react-to-jsx was a great source of inspiration like "I can do this!" :)

### CODE REVIEW

**Comment by alexlande** [comment]
:bow: thanks!

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+**Node from maintainers: You should check [react-element-to-jsx-string](https://github.com/algolia/react-element-to-jsx-string) which does the same thing as react-to-jsx, with more options and updates.**
```
Comment (3rd-Eden) [inline README.md:1]: Note* not node ;-)

Comment (vvo) [reply README.md:1]: Fixing, are you looking for react-element-to-jsx-string? Let me know if you need anything.

Comment (vvo) [reply README.md:1]: Also it's no more the same sentence, it was updated :)

**Pull request merged**